src/holochainClient.js

```diff
@@ -2,7 +2,7 @@ import { connect as hcWebClientConnect } from '@holochain/hc-web-client'
 import { get } from 'lodash/fp'
 import mockCallZome from 'mock-dnas/mockCallZome'
 
-const developmentMockDnaConnection = true // this is the value MOCK_DNA_CONNECTION will have in the dev server
+const developmentMockDnaConnection = false // this is the value MOCK_DNA_CONNECTION will have in the dev server
```

Comment:
should be true on develop

Reply:
So, I just made a script to take care of this issue because I so easily default to testing it live and then pushing the updates, without updating this variable each time. :thinking:  ...But now the script will handle this update for us... so neither of us have to worry about defaulting to the wrong value!! :rocket: